src/Templates/Server/ServerSettings.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+using System.Text;
+using Stl.DependencyInjection;
+
+namespace Samples.Blazor.Server
+{
+    [Settings("Server")]
+    public class ServerSettings
+    {
+        public string GitHubClientId { get; set; } = "7a38bc415f7e1200fee2";
+        public string GitHubClientSecret { get; set; } =
```

Comment:
Put "YOUR_OWN_GITHUB_CLIENT_SECRET" string instead & remove decoding from base64.

Reply:
Let's leave the secret.